test/Unit/SqlInputBindingTests.cs

```diff
@@ -60,17 +61,14 @@ public void TestNullCommand()
         [Fact]
         public void TestNullArgumentsSqlAsyncEnumerableConstructor()
         {
-
             Assert.Throws<ArgumentNullException>(() => new SqlAsyncEnumerable<string>(connection, null));
             Assert.Throws<ArgumentNullException>(() => new SqlAsyncEnumerable<string>(null, new SqlAttribute("")));
         }
 
         [Fact]
-        public void TestNullCurrentValueEnumerator()
+        public void TestInvalidOperationSqlAsyncEnumerableConstructor()
```

Comment:
I'd suggest adding an xml doc comment explaining that this is expected to throw when an invalid/empty connection string setting is given